test/extensions/common/dynamic_forward_proxy/dns_cache_impl_test.cc

```diff
@@ -720,6 +720,34 @@ TEST_F(DnsCacheImplTest, DnsCacheCircuitBreakersOverflow) {
   EXPECT_EQ(1, TestUtility::findCounter(store_, "dns_cache.foo.dns_rq_pending_overflow")->value());
 }
 
+// TEST(DnsCacheImplOptionsTest, UseTcpForDnsLookupsOptionSetOld) {
```

Comment:
these commented tests can be ignored and will be removed line 723 -> 750

Reply:
Why are these being removed?

Reply:
Initially I had just copied these unit tests below them and commented the original. Fixed now.